application/src/@type/cityinfo.tsx

```diff
@@ -0,0 +1,15 @@
+type buildingInfo = {
+  name: string;
+  level: number;
+  built: boolean;
+  // etc.
+};
+
+type cityInfo = {
+  cityHall: buildingInfo;
+  barracks: buildingInfo;
+  forge: buildingInfo;
+  // etc.
+};
+
+export type { buildingInfo, cityInfo };
```

Comment:
remove this file please, it's a duplication of an existing file (`cityinfo.ts`)  and it does not have JSX components (so it should never have had the `.tsx` prefix)